libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -401,3 +393,10 @@ string Parser::expectAsmIdentifier()
 	expectToken(Token::Identifier);
 	return name;
 }
+
+void Parser::expectSinglePusher(dev::solidity::Instruction const& _instruction)
```

Comment:
It does fail on on zero pushers (jump, revert, etc) so removed it from this PR, we can address it separately.